packages/commonwealth/client/scripts/views/modals/WelcomeOnboardModal/WelcomeOnboardModal.tsx

```diff
@@ -60,6 +61,19 @@ const WelcomeOnboardModal = ({ isOpen, onClose }: WelcomeOnboardModalProps) => {
           title: 'Customize your experience',
           component: (
             <PreferencesStep
+              onComplete={() =>
+                setActiveStep(WelcomeOnboardModalSteps.MagicWallet)
+              }
+            />
+          ),
+        };
+      }
+      case WelcomeOnboardModalSteps.MagicWallet: {
+        return {
+          index: 4,
+          title: 'Magic Wallet Creation',
+          component: (
+            <MagicWalletCreationStep
```

Comment:
magic wallets are only made for sms + other sso methods right? If yes shouldn't this be displayed conditionally when a signup with one of those method happens?